app/ts/Elicitation/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const UNRANKED = 0;
```

Comment:
constants.ts is too generic a name. Consider renaming are placing it inside the ts folder if we want one big constants file (I think we don't).

Reply:
rename to elicitationConstants.ts